2.25 eye see tea add attribute values

idelcano opened the pull request:
Closes #84 

You can test it with this malariapp [branch](https://github.com/EyeSeeTea/malariapp/tree/feature-new_sdk_attributevalues)
This branch starts from attributes branch.
this branch only download dataelementattributevalues

xurxodev commented:
@idelcano Threre are a new method pull AttributeValueInteractorImpl but never is invoked, in this branch not download AttributeValue?

idelcano commented:
@xurxodev  thanks, i removed the AttributeValue pull classes because are not necessary.
I updated the malariapp branch too.
The download of attributevalues is inside of each elements(dataelement, program, optionset and organisationunit). (in this branch only dataelementattributevalues)
